assets/prefabs/items/assemblyProcesses/AssembleCrudeShears.prefab

```diff
@@ -0,0 +1,19 @@
+{
+  "Parent": "CrudeAssemblingBaseProcessPart",
+  "ProcessDefinition": {
+    "processType": "ManualLabor:CrudeAssembling"
+  },
+  "RequirementInput": {
+    "requirements": ["CrudeMallet"]
+  },
+  "InventoryInput": {
+    "itemCounts": {
+      "ManualLabor:CrudeShovel": 1
+    }
+  },
```

Comment:
Do you really think this makes sense?
Hitting a crude (stone) shovel with a mallet doesn't sound to me as if anything resembling shears would come from that :thinking:

Reply:
+1